### PR TITLE
Introduce `$permissions` modifier description

### PR DESCRIPTION
**Pull request by sxgunchenko**
https://github.com/AdguardTeam/CoreLibs/issues/419

### CODE REVIEW

**Review by slavaleleka — APPROVED** [review]

File: docs/general/ad-filtering/create-own-filters.md [hunk 1]
```diff
@@ -756,6 +756,41 @@ For the requests matching a `$csp` rule, we will strengthen response security po
...
+
+For the requests matching a `$permissions` rule, AdGuard strengthens response's feature policy by adding additional feature policy equal to the `$permissions` modifier contents. `$permissions` rules are applied independently from any other rule type. Other basic rules have no influence on it **save for document-level exceptions** (see the examples section).
+
+> **Multiple rules matching a single request.**
+>
+> In case if multiple `$permissions` rules match a single request, AdGuard will apply each of them.
+
+**Syntax**
+
+`$permissions` value syntax is similar to the `Permissions-Policy` (or `Feature-Policy`) header syntax.
```
Comment (ameshkov) [inline docs/general/ad-filtering/create-own-filters.md:773]: We need a list of permissions here (at least the most prominent ones + a link to the full list)

**Review by ameshkov — CHANGES_REQUESTED** [review]

**Review by slavaleleka — CHANGES_REQUESTED** [review]
conflicts

File: docs/general/ad-filtering/create-own-filters.md (hunk 1 above)
Comment (sxgunchenko) [reply docs/general/ad-filtering/create-own-filters.md]: Added some more links. Not sure about the most prominent of the directives as most of them are still experimental. And besides, I suppose the list is still subject to change.

File: docs/general/ad-filtering/create-own-filters.md [hunk 2]
```diff
@@ -837,7 +837,8 @@ For the requests matching a `$permissions` rule, AdGuard strengthens response's
 
 **Syntax**
 
-`$permissions` value syntax is similar to the `Permissions-Policy` (or `Feature-Policy`) header syntax.
+`$permissions` value syntax is similar to the `Permissions-Policy` (or `Feature-Policy`) header [syntax](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy).
+See [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy#directives) for the list of the available directives.
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:841]:
```suggestion
The list of the available directives is available [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy#directives).
```

File: docs/general/ad-filtering/create-own-filters.md [hunk 3]
```diff
@@ -823,6 +823,42 @@ For the requests matching a `$csp` rule, we will strengthen response security po
...
+> **Multiple rules matching a single request.**
+>
+> In case if multiple `$permissions` rules match a single request, AdGuard will apply each of them.
+
+**Syntax**
+
+`$permissions` value syntax is similar to the `Permissions-Policy` (or `Feature-Policy`) header [syntax](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy).
+See [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Permissions-Policy#directives) for the list of the available directives.
+
+`$permissions` value can be empty in the case of exception rules. See examples section for further information.
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:843]:
```suggestion
`$permissions` value can be empty in the case of exception rules — see examples below.
```

File: docs/general/ad-filtering/create-own-filters.md [hunk 4]
```diff
@@ -823,6 +823,42 @@ For the requests matching a `$csp` rule, we will strengthen response security po
...
+
+> **Limitations**
+>
+> 1. Please note that there are a few characters forbidden in the `$permissions` value: `,`, `$`;
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:847]:
```suggestion
> 1. Two characters are forbidden in the `$permissions` value: `,` and `$`;
```

File: docs/general/ad-filtering/create-own-filters.md [hunk 5]
```diff
@@ -823,6 +823,42 @@ For the requests matching a `$csp` rule, we will strengthen response security po
...
+> 2. `$permissions` is compatible with the limited list of modifiers: `$domain`, `$important`, `$subdocument`.
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:848]:
```suggestion
> 2. `$permissions` is compatible with the limited list of modifiers: `$domain`, `$important`, and `$subdocument`.
```

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Review by el-termikael — APPROVED** [review]

**Review by slavaleleka — APPROVED** [review]

**Review by ameshkov — APPROVED** [review]

**Pull request merged**